typings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
do we really need typings for zone.js?
According to https://www.npmjs.com/package/@types/zone.js the package already contains type informations.